README.md

```diff
@@ -22,7 +22,7 @@ As part of the audit, Arlo:
 
 Arlo currently supports multiple risk-limiting audit methods, including:
 
-- ballot polling (BRAVO & Minerva)
+- ballot polling (BRAVO, Minerva, and Providence)
```

Comment:
I see later in this PR that Providence and Minerva 2 are used somewhat interchangeably. Could be helpful to either consolidate and consistently use one name or note that they're interchangeable here, e.g. "BRAVO, Minerva, and Providence, i.e. Minerva 2"

Reply:
I'm trying to wipe out references to Minerva2! Oliver Broadrick (one of the authors of the paper) nicely asked me to refer to it as Providence so I'll give it another scan to make sure minerva2 is gone for good. I called it minerva2 since the r2b2 library named it that, but he said that was an artifact of the early days of working on the project 🤷

Reply:
Oh, you can also ignore the minerva2 references in the stub files (anything under `server/stubs/*`. Those are just files autogenerated by a tool in order to help our typing system infer the type of any objects that we import from `r2b2`, so it's up to the maintainers of that library to update their naming scheme.